src/librustc/mir/mod.rs

```diff
@@ -402,9 +402,11 @@ pub enum BorrowKind {
 ///////////////////////////////////////////////////////////////////////////
 // Variables and temps
 
-newtype_index!(Local, "_");
-
-pub const RETURN_POINTER: Local = Local(0);
+newtype_index!(Local,
+    const {
```

Comment:
@nikomatsakis this design is based on a conversation I had with @spastorino. So `MAX` and `DESCRIPTION` are handled as special cases, but all other items in this block are constant. Did I interpret this correctly or was there some other nuance I missed?